website/content/en/docs/best-practices/common-recommendation.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Common recommendations and suggestions"
+linkTitle: "Common suggestions"
+weight: 2
+description: Common recommendations and suggestions to built solutions with Operator SDK
+---
+
+## Overview
+
+Any recommendations or good practices suggested by the Kubernetes community, such as how to to [develop Operator pattern solutions][operator-pattern] or how to [use controller-runtime][controller-runtime] are good recommendations for those who are looking to build operator projects with operator-sdk. Also, see [Operator Best Practices][operator-best-practices]. However, here are some common recommendations.
+
+## Common Recommendations
+
+### Develop idempotent reconciliation solutions
+
+When developing operators, it is important for the controller's reconciliation loop to be idempotent. By following the [Operator pattern][operator-pattern] you will create [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/) which provide a reconcile function responsible for synchronizing resources until the desired state is reached on the cluster.
```

Comment:
woops, think we lost a bit in the shuffle here. Would add a line to the end about why reconciliation being idempotent is important. something like

"As a reconciliation loop can begin with the system in any possible state, it is important for your loop to be idempotent, else the controller or a watched resource maybe become stuck and require manual intervention."

Reply:
I am not sure if I follow your suggestion here.

Reply:
I'd like to have a line that explains WHY is is important for your loop to be idempotent.

Reply:
It is important because of `provide a reconcile function responsible for synchronizing resources until the desired state is reached on the cluster.` If you are not looking for the desired state then, it will not be idempotent imo

Reply:
Many issues scenarios (bad side effects) can be faced if they do not make the impl looking for a final desired state == idempotent since it goes explicitly against its purpose. So, I do not think that we can describe here the possible issues. However, we can put that it will go explicitly against its purpose and the design proposed by controller-runtime. Would that make it better?

Reply:
I think I could address your concern here. Please, let me know wdyt? 
closing that